### PR TITLE
Add Time Relic.app v1.4.3

### PR DESCRIPTION
**Pull request by paracycle**
Time Relic is a new time tracking tool for Mac and Windows

### CODE REVIEW

File: Casks/time-relic.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'time-relic' do
+  version '1.4.3'
+  sha256 'cc96d4f1f0274dc92a8d68d03c7325d1475cc355b4af244f4ba46a6f0784a7e8'
+
+  url 'https://timerelic-download.s3.amazonaws.com/Time%20Relic.zip'
+  name 'Time Relic'
+  homepage 'https:/timerelic.com/'
```
Comment (vitorgalvao) [inline Casks/time-relic.rb]: Not a secure page. It should be `http`. In addition, you’re missing a `/`.

File: Casks/time-relic.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'time-relic' do
+  version '1.4.3'
+  sha256 'cc96d4f1f0274dc92a8d68d03c7325d1475cc355b4af244f4ba46a6f0784a7e8'
+
+  url 'https://timerelic-download.s3.amazonaws.com/Time%20Relic.zip'
```
Comment (vitorgalvao) [inline Casks/time-relic.rb]: Please add a `# amazonaws.com is the official download host per the vendor homepage` comment above `url`.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

File: Casks/time-relic.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name 'Time Relic'
+  homepage 'https:/timerelic.com/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/time-relic.rb]: `:gratis`, it seems.

**Comment by paracycle** [comment]
@vitorgalvao Thanks for all the helpful comments. I've performed the changes you've requested and squashed it down to a single change.

**Comment by vitorgalvao** [comment]
That was quick. Thank you.

**Pull request merged**